NuGet.Config

```diff
@@ -3,5 +3,6 @@
   <packageSources>
     <add key="AspNetVNext" value="https://www.myget.org/F/aspnetcidev/api/v3/index.json" />
     <add key="NuGet" value="https://api.nuget.org/v3/index.json" />
+    <add key="NuGetBuild" value="https://www.myget.org/F/nugetbuild/api/v3/index.json" />
```

Comment:
We have a copy of their binaries in our feeds already. You probably don't need this. https://github.com/aspnet/CoreCLR/blob/dev/tools/MirrorLatestCoreCLR/Program.cs#L97